setup.py

```diff
@@ -46,6 +49,7 @@ def __str__(self):
 setup(
     name="katsdpimager",
     description="GPU-accelerated radio-astronomy spectral line imager",
+    long_description=long_description,
```

Comment:
In order to add `README.md` content on PyPi properlly, Perhaps also add the [description-content-type](https://packaging.python.org/specifications/core-metadata/#description-content-type).

See: https://packaging.python.org/guides/making-a-pypi-friendly-readme/#including-your-readme-in-your-package-s-metadata

Reply:
The default content type for long_description is reST, so it's not necessary to specify unless you're using a different content type like markdown.

Reply:
Learn something new every day. :wink: